pkg/apis/deployment/volumemount.go

```diff
@@ -176,11 +178,13 @@ func GetCsiAzureVolumeMountType(radixVolumeMount *radixv1.RadixVolumeMount) radi
 
 func getCsiRadixVolumeTypeIdForName(radixVolumeMount *radixv1.RadixVolumeMount) (string, error) {
 	if radixVolumeMount.BlobFuse2 != nil {
-		switch radixVolumeMount.BlobFuse2.Protocol {
-		case radixv1.BlobFuse2ProtocolFuse2:
+		switch string(radixVolumeMount.BlobFuse2.Protocol) {
+		case string(radixv1.BlobFuse2ProtocolFuse2), "":
 			return "csi-blobfuse2-fuse2", nil
-		case radixv1.BlobFuse2ProtocolNfs:
+		case string(radixv1.BlobFuse2ProtocolNfs):
 			return "csi-blobfuse2-nfs", nil
+		default:
+			return "", fmt.Errorf("unknown blobfuse2 protocol %s", radixVolumeMount.BlobFuse2.Protocol)
 		}
```

Comment:
```suggestion
		switch radixVolumeMount.BlobFuse2.Protocol {
		case radixv1.BlobFuse2ProtocolFuse2, "":
			return "csi-blobfuse2-fuse2", nil
		case radixv1.BlobFuse2ProtocolNfs:
			return "csi-blobfuse2-nfs", nil
		default:
			return "", fmt.Errorf("unknown blobfuse2 protocol %s", radixVolumeMount.BlobFuse2.Protocol)
		}
```
Type conversion to string is not required

Reply:
Fixed